relay/pkg/block/push.go

```diff
@@ -70,15 +70,42 @@ func (f *Forwarder) pushHeadersToHostChain(
 
 	if startMod == 0 {
 		// we have a difficulty change first
-		// TODO: implementation
+		logger.Infof(
```

Comment:
If we don't use any arguments to logging message, then perhaps `logger.Info` ?

Reply:
See https://github.com/keep-network/tbtc/commit/ed0fb5df014da81a12118fd087306d8d66528ec5